docs/docs/assets-and-files.md

```diff
@@ -102,3 +103,66 @@ const Header = () => {
 
 export default Header
 ```
+
+## Styling SVGs: The special type of image
+
+By default you can import and use SVG images like any other image asset.
+
+```jsx title="web/src/components/Example.js"
+// highlight-next-line
+import svgIconSrc from '../mySvg.svg'
+
+const Example = () => {
+  return (
+    <>
+      // highlight-next-line
+      <img src={svgIconSrc} alt="Logo" />
+    </>
+  )
+}
+
+export default Example
+```
+
+Sometimes however, you might want more control over styling your SVGs - maybe you want to modify the `stroke-width` or `fill` color.
+
+The easiest way to achieve this, is to make your SVGs a React component. Open up your SVG file, and drop in it's contents into a component - for example:
+
+```tsx title="web/src/components/icons/CarIcon.tsx"
+import type { SVGProps } from "react"
+
+export const CarIcon = (props: SVGProps) => {
+  return (
+// 👇 content of your SVG file
+    <svg
+      // highlight-next-line
+      className="fill-blue-500" // 👈 you can use classes, like with tailwind
+      // highlight-next-line
+      stroke={props.strokeColor} // or adjust properties directly
+    // ...
+```
+
+If you needed to convert a whole library of SVGs into stylable (or animatable!) components, one easy way would be to use the [SVGR cli](https://react-svgr.com/docs/cli/)
+
+
+
+## Custom fonts
+There's many different ways to peel this potato - it's all a search away - but if you're using the CSS `@font-face` rule, we have a quick tip for you:
+
+1. Place your fonts in the public folder, so it gets carried across
+2. In your CSS, use relative paths to point to the font file, for example:
+
+```css
+/* in some CSS file you are loading in your project */
+@font-face {
+    font-family: 'Redwood Neue';
+    /* 👇 it's a relative path */
+    // highlight-next-line
+    src: url('../../public/fonts/RedwoodNeue.woff2')
+      format('woff2');
+    font-weight: 300;
+    font-style: italic;
+    ascent-override: 97%;
+  }
+```
+This will make sure the that fonts are being loaded correctly across your dev server and storybook - where there are subtle differences in how paths are processed.
```

Comment:
```suggestion
This will make sure that the fonts are being loaded correctly across your dev server and storybook – where there are subtle differences in how paths are processed.
```